script.js

```diff
@@ -1,10 +1,12 @@
 function update(){
     let time=new Date();
     let hrs=time.getHours();
+    suffix = hrs >= 12 ? "PM":"AM";
+    hrs=((hrs % 12) || 12);
     document.getElementById('hours').innerHTML=hrs;
```

Comment:
declare the variables always with something like let or var, this is JS, not python.